public/js/app/components/MemoMenu.jsx

```diff
@@ -30,9 +30,9 @@ function MemoMenu(props) {
       cancel: false,
     },
     semester: initSemester && initSemester.length > 0 ? initSemester : '',
-    rounds: initTempData ? initTempData.roundIdList : [],
-    usedRounds: context.usedRounds.usedRoundsIdList || [],
-    usedRoundsWithWebVer: context.usedRounds.roundsIdWithWebVersion || {},
+    rounds: initTempData ? initTempData.applicationCodes : [],
+    usedRounds: context.usedRounds.usedRoundsApplicationCodeList || [],
```

Comment:
Is `kurs-pm-data-api` reflect these name changes in `usedRounds`? Will it have `usedRoundsApplicationCodeList` and `roundsApplicationCodeWithWebVersion`?

Reply:
This data is coming from `kurs-pm-api`. There it is updated